File: lumira-app/src/data/templates/anime-dream-portrait.ts
import type { PhotoTemplate } from '@/types/template'

// 模板：动漫温柔青 — 宫崎骏感饱和提亮梦境青，晴天草地张开双臂的动漫浪漫
const animeDreamPortrait: PhotoTemplate = {
  meta: {
    id: 'anime_dream_portrait',
    name: '动漫温柔青',
    author: '如画 Lumira',
    version: '1.0.0',
    category: 'portrait',
    classification: { type: 'portrait', style: 'anime_dream', method: 'full_body' },
    tags: ['人像', '动漫', '宫崎骏', '梦境', '温柔青'],
    tagIds: [],
    price: 3,
    cover: 'https://picsum.photos/seed/template-anime-dream-portrait/400/600',
    description: '宫崎骏感饱和提亮梦境青，晴天草地张开双臂的动漫浪漫',
    referenceSource: '设计规范 v1.0 模板 12；参考小红书梦境滤镜教程 / 宫崎骏动漫感'
  },
  composition: {
    overlayType: 'rule_of_thirds',
    subjectFrame: { x: 0.2, y: 0.1, w: 0.55, h: 0.8 },
    opacity: 0.2,
    aspectRatio: '3:4',
    description: '三分线左侧偏中构图，全身取景，天空留白'
  },
  pose: {
    silhouette: { type: 'builtin', data: 'anime-dream-portrait' },
    position: { x: 0.5, y: 0.5 },
    scale: 0.7,
    rotation: 0,
    description: '正面站立张开双臂，仰头看天，微张站立，裙摆飘动，开心大笑'
  },
  camera: {
    exposureCompensation: 0,
    iso: 100,
    shutterSpeed: '1/200',
    whiteBalance: 'daylight',
    whiteBalanceK: 5500,
    flashMode: 'off',
    focusMode: 'auto',
    filterPreset: 'none',
    lensType: '1x'
  },
  sceneGuide: {
    lightDirection: '顺光 / 顶光',
    lightDirectionAngle: 30,
    shootingDistance: '2-3m',
    background: '蓝天白云 / 草地 / 花海',
    props: [],
    bestTime: '上午 9:00-11:00',
    bestTimeFrom: '09:00',
    bestTimeTo: '11:00',
    tips: [
      '晴天蓝天才有效果',
      '仰拍带天空',
      '服装浅色与天空呼应'
    ]
  },
  postProcess: {
    cropRatio: '3:4',
    color: {
      brightness: 12,
      contrast: -5,
      saturation: 10,
      temperature: 5,
      tint: 0,
      highlights: -5,
      shadows: 15,
      clarity: -5,
      vibrance: 8
    },
    smoothStrength: 10,
    sharpen: 5,
    vignette: 0,
    grain: 0,
    lut: 'pastel'
  }
}

export default animeDreamPortrait

File: lumira_app_flutter/lib/features/capture/data/templates/anime_dream_portrait.dart
// lib/features/capture/data/templates/anime_dream_portrait.dart
import '../../domain/photo_template.dart';
import '../../../templates/data/remote_template_dto.dart';

/// 动漫温柔青模板（anime_tender 子风格，归位 fresh_healing）
/// 来源：docs/superpowers/specs/2026-08-04-portrait-template-redesign-design.md 模板 12
const PhotoTemplate animeDreamPortraitTemplate = PhotoTemplate(
  meta: TemplateMeta(
    id: 'anime_dream_portrait',
    name: '草地蓝天跳跃少女｜清新日系人像模板',
    author: 'Lumira',
    version: '1.0.0',
    category: 'portrait',
    classification: TemplateClassification(type: 'portrait', majorStyle: 'fresh_healing', style: 'anime_tender', subStyle: 'anime_tender', method: 'full_body'),
    tags: ['人像', '日系清新', '草坪', '蓝天', '跳跃抓拍', '少女感', '户外'],
    tagIds: [],
    price: 20,
    images: [
      TemplateImage(url: 'assets/images/templates/anime_dream_portrait.jpg'),
    ],
    description: '户外晴天草坪人像模板，低机位仰拍捕捉背对镜头跳跃瞬间，利用澄澈蓝天大留白，搭配浅色系洛丽塔风裙装，适合夏日晴天拍摄元气自由感的少女写真。',
    referenceSource: '小红书梦境滤镜教程；宫崎骏动漫感；晴天户外人像',
    shortDesc: '向着蓝天白云纵身跃起，满是自由元气的夏日氛围感☁️',
    ambience: RemoteTemplateAmbienceDto(
      seasons: ['summer'],
      weathers: ['sunny'],
      timeTones: ['day'],
    ),
  ),
  composition: Composition(
    overlayType: 'rule_of_thirds',
    gridType: 'rule_of_thirds',
    subjectFrame: SubjectFrame(x: 0.42, y: 0.32, w: 0.34, h: 0.56),
    opacity: 0.3,
    aspectRatio: '9:16',
    description: '竖版9:16构图，下方保留绿色草地，上方大面积留给蓝天白云，人物主体放在画面中间区域，天空做大量留白，强化开阔自由的感觉。',
  ),
    poses: [
    Pose(
      silhouette: SilhouetteResource(type: 'image', data: 'assets/images/silhouettes/anime_dream_portrait.png'),
          position: Position(x: 0.5, y: 0.5),
          scale: 2.5,
          rotation: 0,
          description: '人物背对镜头，双脚腾空跳跃，屏幕左侧手臂竖直向上高举握拳，屏幕右侧手臂向身体侧后方张开握拳，双腿屈膝，一条腿向上抬，一条腿向下，呈现起跳腾空姿态。',
          cameraDirection: 'back',
    ),
  ],
  camera: CameraParams(
    exposureCompensation: 0.2,
    iso: 100,
    shutterSpeed: '1/200',
    whiteBalance: 'daylight',
    whiteBalanceK: 5700,
    flashMode: 'off',
    focusMode: 'continuous',
    lensType: '1x',
    lensSuggestion: 'main',
  ),
  sceneGuide: SceneGuide(
    lightDirection: '晴天白天自然光，光源屏幕右上方45°，软日光，低光比',
    lightDirectionAngle: 45,
    shootingDistance: '2.2-2.8m',
    background: '前景绿色起伏草坪，背景布满蓬松白云的湛蓝色天空，无多余杂物',
    props: ['装饰花草草帽'],
    bestTime: '14:00-16:00',
    bestTimeFrom: '14:00',
    bestTimeTo: '16:00',
    tips: [
      '拍摄者放低手机做低机位仰拍，镜头朝向斜上方',
      '让模特多尝试连续跳跃，抓拍腾空最高点瞬间',
      '尽量找干净无杂物的草坪，避开地面多余垃圾物体',
      '选择云朵丰富的晴天，蓝天氛围感更强',
    ],
  ),
  postProcess: PostProcess(
    cropRatio: '9:16',
    color: PostProcessColor(
      brightness: 4,
      contrast: -6,
      saturation: 8,
      temperature: -4,
      tint: 2,
      highlights: -12,
      shadows: 10,
      clarity: -5,
      vibrance: 12,
    ),
    smoothStrength: 25,
    sharpen: 12,
    vignette: 18,
    grain: 22,
    lut: 'japanese_fresh',
  ),
);
